lib/bundler/cli/list.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Bundler
+  class CLI::List
+    def run
+      specs = Bundler.load.specs.reject {|s| s.name == "bundler" }
+
+      return Bundler.ui.info "No gems in the Gemfile" if specs.empty?
+      Bundler.ui.info "Gems included by the bundle:"
+      specs.sort_by(&:name).each do |s|
+        Bundler.ui.info "  * #{s.name} (#{s.version}#{s.git_version})"
```

Comment:
gem name only with version on verbose was my first thought, but then I checked npm and yarn, and they both always include the version... I guess is there ever a time that it would be useful to have the names without also having the versions? scripting? should we provide `bundle ls --names-only` for that?